src/util/getReceiverAddress.ts

```diff
@@ -25,21 +26,38 @@ interface Props {
 const getReceiverAddress = ({ receiverAddress, chain }: Props) => {
   const chainsConfigs = getChainConfigs();
   const { SS58Format, bridgeId } = chainsConfigs[chain];
-  try {
-    const [validatedDerivedAcccount] = checkAddress(receiverAddress, SS58Format);
-    let address = receiverAddress;
-    if (!validatedDerivedAcccount) {
-      const formattedAccount = getDeriveAccount({
-        SS58Format,
-        address: receiverAddress,
-        bridgeId
-      });
 
-      address = formattedAccount;
+  try {
+    const [validatedDerivedAcccount, rest] = checkAddress(receiverAddress, SS58Format);
+    if (validatedDerivedAcccount) {
+      return { address: receiverAddress, formatFound: chain };
     }
-    return address;
+    // should be extracted as a separate component/function
+    const getFormat = (prefix: string) => {
+      const intPrefix: number = parseInt(prefix, 10);
+      if (intPrefix === GENERIC_SUBSTRATE_PREFIX /* 42 */) {
```

Comment:
```suggestion
      if (intPrefix === GENERIC_SUBSTRATE_PREFIX) {
```
I was just suggesting that the constant should contain that value.